fedocal/templates/default/master.html

```diff
@@ -98,6 +98,7 @@ <h4 class="hide">{{ _('Main Site Links') }}</h4>
             filename='jquery-ui.min-1.11.1.js') }}">
     </script>
     <script type="text/javascript">
+        FEDOCAL = {};
```

Comment:
@pypingou the thing is that only the modal template extends ``base_partial.html``. In the ``view_meeting.html`` template, there's still an ``if`` condition to decide which template to extend this template from. So, if I moved the above initialization of ``FEDOCAL`` var to ``base_partial`` template, this variable won't be available in meeting details page. However, doing it in the current way ensures that the variable is properly defined and accessible in ``full`` and ``partial`` templates, because partial templates are always loaded on top of full templates.